lib/ramble/ramble/application.py

```diff
@@ -2437,6 +2437,7 @@ def _object_templates(self, workspace):
         run_dir = self.expander.experiment_run_dir
         replacements = workspace.workspace_paths()
         expander = self.expander
+        tpl_ext = ".tpl"
```

Comment:
I kind of think this extensions should live somewhere else. We use it in the workspace as well, so that should probably use the same extension variable.